### PR TITLE
doc: add adopter list

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: Feynman Zhou <feynmanzhou@microsoft.com>

### CODE REVIEW

File: content/en/docs/adopters.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+We are happy and proud to have you all as part of the community! To add your use case and organization to this list, please raise a pull request.
+
+## Notation Adopters
+
+All organizations are sorted alphabetically below.
+
+| Organization | Website | Scenario | 
+| --- | --- | --- | 
+| Amazon | https://aws.amazon.com/ | AWS container signing solution |
```
Comment (FeynmanZhou) [inline content/en/docs/adopters.md:18]: @vaninrao10 Could you pls help to confirm and provide the scenario in Amazon?

**Review by yizha1 — DISMISSED** [review]
LGTM

**Comment by yizha1** [comment]
@FeynmanZhou  how do we proceed with this PR?

File: content/en/docs/adopters.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Notation Adopters"
+description: "List the known adopters who use Notation in production or integrate Notation into their products and services."
```
Comment (zr-msft) [inline content/en/docs/adopters.md:3]:
```suggestion
description: "Known adopters who use Notation in production or integrate Notation into their products and services."
```

File: content/en/docs/adopters.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+type: docs
+weight: 9
+---
+
+The following organizations are known adopters who use Notation in production or integrate Notation into their products and services.
+
+We are happy and proud to have you all as part of the community! To add your use case and organization to this list, please raise a pull request.
+
+## Notation Adopters
```
Comment (zr-msft) [inline content/en/docs/adopters.md:12]:
```suggestion

```

Given the scope, i don't think you need this header

File: content/en/docs/adopters.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+All organizations are sorted alphabetically below.
+
+| Organization | Website | Scenario | 
+| --- | --- | --- | 
+| Amazon | https://aws.amazon.com/ | AWS container signing solution |
+| Docker | https://www.docker.com/ | Sign and verify Docker container images |
+| Microsoft | https://www.microsoft.com/ | Sign and verify images published by Microsoft in MAR |
```
Comment (zr-msft) [inline content/en/docs/adopters.md:20]:
```suggestion
| Microsoft | https://www.microsoft.com/ | [Build, sign, and verify container images using Notary and Azure Key Vault](https://learn.microsoft.com/azure/container-registry/container-registry-tutorial-sign-build-push?wt.mc_id=azurelearn_inproduct_oss_notaryproject) |
```

Linking to relevant docs

**Review by zr-msft — DISMISSED** [review]
Thank you @FeynmanZhou 

Also, i think this would make more sense in the `concepts` directory as apposed to the top-level

**Comment by zr-msft** [comment]
@FeynmanZhou and @toddysm is this still a relevant PR, or can it be closed?

File: content/en/docs/adopters.md (hunk 1 above)
Comment (FeynmanZhou) [reply content/en/docs/adopters.md]: @iamsamirzon @priteshbandi Could you pls help to confirm and provide the scenario at Amazon? I added AWS Signer to this list but you can change it as you want.

File: content/en/docs/adopters.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+We are happy and proud to have you all as part of the community! To add your use case and organization to this list, please raise a pull request.
+
+All organizations are sorted alphabetically below.
+
+| Organization | Website | Scenario | 
+| --- | --- | --- | 
+| Amazon | https://aws.amazon.com/ | [AWS Signer](https://docs.aws.amazon.com/signer/latest/developerguide/image-signing-prerequisites.html) |
+| containerd |  http://containerd.io/ |  [container image sign and verify with notation tool](https://github.com/containerd/nerdctl/blob/main/docs/notation.md) |
+| Harbor | https://goharbor.io/ |  Storage and display Notary Project signature |
```
Comment (yizha1) [inline content/en/docs/adopters.md:18]: I found this link with Notation https://goharbor.io/docs/main/working-with-projects/working-with-images/sign-images/, but it contains information with cosign

**Review by yizha1 — APPROVED** [review]
LGTM with a suggestion

File: content/en/docs/adopters.md (hunk 5 above)
Comment (FeynmanZhou) [reply content/en/docs/adopters.md]: Updated

**Review by yizha1 — APPROVED** [review]
LGTM

File: content/en/docs/adopters.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+All organizations are sorted alphabetically below.
+
+| Organization | Website | Scenario | 
+| --- | --- | --- | 
+| Amazon | https://aws.amazon.com/ | [AWS Signer](https://docs.aws.amazon.com/signer/latest/developerguide/image-signing-prerequisites.html) |
+| containerd |  http://containerd.io/ |  [container image sign and verify with notation tool](https://github.com/containerd/nerdctl/blob/main/docs/notation.md) |
+| Harbor | https://goharbor.io/ |  [Storage and display Notary Project signature](https://goharbor.io/docs/main/working-with-projects/working-with-images/sign-images/#use-notationexperimental-to-sign-artifacts-with-distribution-spec-v11-mode) |
+| Kyverno | https://kyverno.io/ | [Support verifying Notary Project signature](https://kyverno.io/docs/writing-policies/verify-images/notary/) |
+| Microsoft | https://www.microsoft.com/ | [Build, sign, and verify container images using Notation and Azure Key Vault](https://learn.microsoft.com/azure/container-registry/container-registry-tutorial-sign-build-push?wt.mc_id=azurelearn_inproduct_oss_notaryproject)  |
+| Ratify |  https://ratify.sh/ | Validating OCI Artifact signatures using Notation library |
```
Comment (priteshbandi) [inline content/en/docs/adopters.md:21]: Are we missing hyper link for `Validating OCI Artifact signatures using Notation library`?

Comment (FeynmanZhou) [reply content/en/docs/adopters.md:21]: @priteshbandi  The Ratify website link is enough for this case.

**Pull request merged**